x/posts/spec/02-concepts.md

```diff
@@ -0,0 +1,202 @@
+
+##### Url
+The actual value of the url where the user should be redirected to.
+
+##### Display url (Optional)
+The visual url that should be displayed in the app. Ideally something that shorten or make an url more human-readable.
+
+### Author
+The address of the author of the post.
+
+### Conversation ID (optional)
+The ID of the original post of the conversation. This ID identifies the parent post (if existent) where the conversation
+started.
+
+### Referenced Posts
+Referenced posts are represented by a `PostReference` structure.
+
+#### PostReference
+A reference to an external post, usable for reply, repost or quote purposes.
+
+##### Type
+This is the type of the reference. It can be one of the following values:
+
+| **Name**                          | **Description**                                          |  
+|:----------------------------------|:---------------------------------------------------------|
+| `POST_REFERENCE_TYPE_UNSPECIFIED` | No reference specified                                   |
+| `POST_REFERENCE_TYPE_REPLY`       | This reference represents a reply to the specified post  |
+| `POST_REFERENCE_TYPE_QUOTE`       | This reference represents a quote of the specified post  |
+| `POST_REFERENCE_TYPE_REPOST`      | This reference represents a repost of the specified post |
+
+##### Post ID
+The ID of the referenced post.
+
+##### Position
+This field should be used only when the `Type` field is equal to `TYPE_QUOTE` to indicates where the `PostReference`
+start in the post's text.
+
+### Reply Setting
+This field contains the possible reply settings that a post can have. It can be one of the following values:
+
+| **Name**                    | **Description**                                              |  
+|:----------------------------|:-------------------------------------------------------------|
+| `REPLY_SETTING_UNSPECIFIED` | No reference specified                                       |
+| `REPLY_SETTING_EVERYONE`    | This reference represents a reply to the specified post      |
+| `REPLY_SETTING_FOLLOWERS`   | This reference represents a quote of the specified post      |
+| `REPLY_SETTING_MUTUAL`      | This reference represents a repost of the specified post     |
+| `REPLY_SETTING_MENTIONS`    | Only people mentioned inside this post will be able to reply |
+
+### Creation Date
+The creation date of the post.
+
+### Last Edited Date
+The las time the post has been edited.
+
+## Attachment
+An attachment represent something that can be added to a post in order to enrich it or give it some additional use cases.
+
+### Subspace ID
+The [subspace] ID indicates the ID of the Dapp where the attachment is hosted and lives.
+
+### Post ID
+The [post](#Post) ID to which the attachment is linked.
+
+### ID
+The unique ID that identifies the attachment. This ID is automatically assigned the same way the post one is.
```

Comment:
```suggestion
An attachment id is the unique identifier within a specific post. This is automatically created when handling a `MsgCreatePost` or `MsgAddPostAttachment`. With the subspace id and post id, it can be used to uniquely identify an attachment within Desmos.
```